README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
+- No ads, trackers, subscriptions, etc.
```

Comment:
For the subscriptions part, I would add: "for anything that the app does locally". If we want to create a cloud-based service to keep your data, for example, that could be subscription-based.

Reply:
Updated